services/app/app/controllers/portal/leadership.js

```diff
@@ -8,24 +8,41 @@ export default Controller.extend({
   isModalOpen: false,
   hash: null,
 
-  submitDisabled: computed('categories.[]', function() {
-    return this.get('categories.length') === 0;
+  submitDisabled: computed('payload.{added,removed}', function() {
+    const { added, removed } = this.get('payload');
+    return added.length === 0 && removed.length === 0;
+  }),
+
+  payload: computed('selected.[]', 'initial.[]', 'config.leadershipAllowCategoryRemoval', function() {
+    const sectionRemovalAllowed = this.get('config.leadershipAllowCategoryRemoval');
+    const { selected, initial } = this.getProperties('selected', 'initial');
+    const payload = { removed: [], added: [] };
+    selected.forEach((id) => {
+      if (!initial.includes(id)) payload.added.push(id);
+    });
+    if (sectionRemovalAllowed) {
+      initial.forEach((id) => {
+        if (!selected.includes(id)) payload.removed.push(id);
+      });
+    }
+    return payload;
   }),
 
   categoryPrefix: computed.reads('config.leadershipCategoryPrefix'),
 
   init() {
     this._super(...arguments);
-    this.set('categories', []);
+    this.set('selected', []);
+    this.set('initial', []);
   },
 
   actions: {
     update(id) {
-      const categories = this.get('categories');
-      if (categories.includes(id)) {
-        categories.removeObject(id);
+      const selected = this.get('selected');
+      if (selected.includes(id)) {
+        selected.removeObject(id);
```

Comment:
This should only happen if category removal is enabled -- the category shouldn't be removable in the UI if it's not going to be allowed in the submission.